command/login/login.go

```diff
@@ -36,6 +36,8 @@ type cmd struct {
 	tokenSinkFile   string
 	meta            map[string]string
 
+	aws *AWSLogin
```

Comment:
Why make this a pointer if it's always initialized? You could just skip line 64 and make this `aws AWSLogin`